profile/README.md

```diff
@@ -3,77 +3,59 @@
 <p align="center">
 <img src="https://github.com/instructlab/.github/blob/main/assets/instructlab-banner.png">
 </p>
```

Comment:
```suggestion
![Banner](https://github.com/instructlab/.github/blob/main/assets/instructlab-banner.png)
```

With this nice banner, I don't think it's necessary to do the custom HTML anymore.